spanner/spanner_snippets/spanner/spanner_max_commit_delay.go

```diff
@@ -25,22 +25,15 @@ import (
 	"cloud.google.com/go/spanner"
 )
 
-// maxCommitDelay sets the maximum commit delay for a transaction.
-func maxCommitDelay(w io.Writer, db string) error {
-	// db = `projects/<project>/instances/<instance-id>/database/<database-id>`
+func setMaxCommitDelay(w io.Writer, db string) error {
+	// db is the fully-qualified database name of the form `projects/<project>/instances/<instance-id>/database/<database-id>`
```

Comment:
add example value to be consistent with other samples
`// db := "projects/my-project/instances/my-instance/databases/my-database"`

https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#function-arguments-for-snippets